src/Components/Pages/MissionsPage.js

```diff
@@ -1,5 +1,69 @@
-const Missions = () => (
-  <h1>Missions</h1>
-);
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import '../../App.css';
+import { fetchMissions, reserveMission } from '../../Redux/mission/mission';
+
+const Missions = () => {
+  const missions = useSelector((state) => state.missionReducer);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (missions.length === 0) {
+      dispatch(fetchMissions());
+    }
+  }, [dispatch, missions.length]);
+  const clickHandler = (event) => {
+    dispatch(reserveMission(event.target.id));
+  };
+
+  return (
+    <table className="wrapper">
+      <thead>
+        <tr>
+          <th>Mission</th>
+          <th className="description">Description</th>
+          <th>Status</th>
+        </tr>
+      </thead>
+      <tbody>
+        {missions.map((item) => (
+          <tr key={item.id} className="tr-row">
+            <td className="mission-name">{item.name}</td>
+            <td>{item.description}</td>
+            <td>
+              {item.reserved ? (
+                <button type="button" className="member">ACTIVE MEMBER</button>
```

Comment:
I think you should change to `<span>` or other tags instead of `<button>` tag becuae it is clickabe in browser. `Active member` text is not clickable

Reply:
Sure, thanks 👍